tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -1782,7 +1782,7 @@ def test_org_auth_token(self):
             good_token_str = generate_token(org.slug, "")
             OrgAuthToken.objects.create(
                 organization_id=org.id,
-                name="token 1",
+                name="token 2",
                 token_hashed=hash_token(good_token_str),
                 token_last_characters="ABCD",
                 scope_list=["org:ci"],
```

Comment:
Why are these changes needed? And, should we perhaps add a test for the caching functionality?

Reply:
There were two tokens with a name of 'token 1' for the same org which made debugging harder.